inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -256,6 +256,43 @@ class V10Parser : public IParser {
             stringToType<double>(val, value);
             adapter.set(value);
         }
+        void on_adapter(const std::string& name, ngraph::ValueAccessor<void*>& adapter) override  {
+            std::string value;
+            pugi::xml_node dn = node.child("data");
+            auto type = XMLParseUtils::GetStrAttr(node, "type");
+
+            if (dn.empty())
+                THROW_IE_EXCEPTION << "No attrtibutes defined for " << type << " op!";
+
+            if (getStrAttribute(dn, name, value)) {
+                auto data = static_cast<char*>(adapter.get_ptr());
+                size_t length = std::min(value.size(), adapter.size());
+                value.copy(data, length);
+            } else if (name == "value") {
```

Comment:
I propose to use next conditional: `name == "value" && type == "constant"`

Reply:
Corrected, introduced new condition.